src/services/marketplace.ts

```diff
@@ -54,7 +54,7 @@ export const getProfileImage = async (username: string) => {
     throw new Error('No access token found');
   }
 
-  const response = await fetch(`/api/profiles/image/${username}`, {
+  const response = await fetch(`api/profiles/image/${username}`, {
```

Comment:
Why are you using raw `fetch` instead of the wrapped `http` service? This is not aligned with the rest of the codebase, and causes issues like this that have already been solved in `http`. This also makes long-term maintenance a pain because our HTTP code is now spread around the codebase instead of centralized in the `http` service like it's supposed to be.

Reply:
More to the point, why did you create new `getProfileInfo` and `getProfileImage` methods in the marketplace service, instead of using the existing ones in `user.ts`?